pkg/pricer/pricer.go

```diff
@@ -39,3 +41,8 @@ func (pricer *FixedPricer) PeerPrice(peer, chunk swarm.Address) uint64 {
 func (pricer *FixedPricer) Price(chunk swarm.Address) uint64 {
 	return pricer.PeerPrice(pricer.overlay, chunk)
 }
+
+func (pricer *FixedPricer) MostExpensive() *big.Int {
+	mostExp := int64(10 * uint64(swarm.MaxPO) * pricer.poPrice)
```

Comment:
to avoid overflow of the multiplication, you can do the calculation like 
new(big.Int).Mul( new(big.Int).Mul(big.NewInt(10), new(big.Int).SetUint64(uint64(swarm.MaxPO)), new(big.Int).SetUint64(pricer.poPrice) )

Reply:
thanks, I'll use this one instead